[2022-09-09T23:25:11.505+0600] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_task_v2.second_task scheduled__2021-08-03T00:00:00+00:00 [queued]>
[2022-09-09T23:25:11.508+0600] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_task_v2.second_task scheduled__2021-08-03T00:00:00+00:00 [queued]>
[2022-09-09T23:25:11.508+0600] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-09T23:25:11.508+0600] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-09-09T23:25:11.508+0600] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-09T23:25:11.525+0600] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): second_task> on 2021-08-03 00:00:00+00:00
[2022-09-09T23:25:11.527+0600] {standard_task_runner.py:52} INFO - Started process 4622 to run task
[2022-09-09T23:25:11.529+0600] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'first_task_v2', 'second_task', 'scheduled__2021-08-03T00:00:00+00:00', '--job-id', '338', '--raw', '--subdir', '/home/mahim-ubuntu/home/python/airflow-project/airflow/lib/python3.10/site-packages/airflow/example_dags/testDags.py', '--cfg-path', '/tmp/tmp3ts0a8d0', '--error-file', '/tmp/tmp7mfy4vfy']
[2022-09-09T23:25:11.530+0600] {standard_task_runner.py:80} INFO - Job 338: Subtask second_task
[2022-09-09T23:25:11.565+0600] {task_command.py:371} INFO - Running <TaskInstance: first_task_v2.second_task scheduled__2021-08-03T00:00:00+00:00 [running]> on host MahimPC.
[2022-09-09T23:25:11.612+0600] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=clickSpikes
AIRFLOW_CTX_DAG_ID=first_task_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-03T00:00:00+00:00
[2022-09-09T23:25:11.613+0600] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-09-09T23:25:11.613+0600] {subprocess.py:74} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am Mahim. How are you?']
[2022-09-09T23:25:11.616+0600] {subprocess.py:85} INFO - Output:
[2022-09-09T23:25:11.617+0600] {subprocess.py:92} INFO - hey, I am Mahim. How are you?
[2022-09-09T23:25:11.617+0600] {subprocess.py:96} INFO - Command exited with return code 0
[2022-09-09T23:25:11.639+0600] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=first_task_v2, task_id=second_task, execution_date=20210803T000000, start_date=20220909T172511, end_date=20220909T172511
[2022-09-09T23:25:11.661+0600] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-09T23:25:11.672+0600] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
